src/components/SipProvider/index.js

```diff
@@ -264,12 +264,7 @@ export default class SipProvider extends React.Component {
     }
 
     try {
-      let wssUri = `wss://${host}:${port}`;
-      if (path) {
-        wssUri += `/${path}`;
-      }
-
-      const socket = new JsSIP.WebSocketInterface(wssUri);
+      const socket = new JsSIP.WebSocketInterface(`wss://${host}:${port}${pathname}`);
```

Comment:
This will probably become `wss://sip.example.com:7443undefined` if pathname is not set. Something like `wss://${host}:${port}${pathname || ''}` might be necessary

Reply:
@kachkaev fixed.